# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: js/src/jit-test/tests/warp/bug1668197.js
# Commit: cb4534b7dbb7
# Full Hash: cb4534b7dbb746d083bdbf8fb74b3036c63f897b
# Author: Iain Ireland <iireland@mozilla.com>
# Date: 2020-10-02 04:10:47
# Description:
#   Bug 1668197: Handle OOM in MHypot::New r=jandem
#   
#   In addition to the calls in the transpiler, MHypot::New is also called in the MHypot `clone` implementation.
#   
#   I checked the other fallible `New` methods, and it looks like we are handling everything else correctly.
# ==============================================================================

diff -r 66666ee5b0a9 -r cb4534b7dbb7 js/src/jit-test/tests/warp/bug1668197.js
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/js/src/jit-test/tests/warp/bug1668197.js	Thu Oct 01 16:32:15 2020 +0000
@@ -0,0 +1,6 @@
+// |jit-test| skip-if: !('oomTest' in this)
+function f(x, y) {
+    return ~Math.hypot(x >>> 0, 2 - x >>> 0);
+}
+f(2, Math);
+oomTest(f);